client/main.lua

```diff
@@ -471,6 +482,9 @@ if cfg.torqueMultiplierEnabled or cfg.preventVehicleFlip or cfg.limpMode then
             if cfg.preventVehicleFlip then
                 preventVehicleFlip()
             end
+            if cfg.preventAirControl then
+                preventAirControl()
+            end
         end
     end)
 end
```

Comment:
This is missing the check to put preventAirControl into starting the thread. Additionally if you could move this into a thread that only runs when in a vehicle that can improve performance.

Reply:
This was supposed to be at line 473